serverless.yml

```diff
@@ -29,11 +29,10 @@ custom:
 functions:
     main:
         name: ${self:service}-main
-        timeout: 300
+        timeout: 290 # timeout before next lambda execution to prevent concurrency
         handler: build/index.main
-        reservedConcurrency: 1
```

Comment:
It requires an aws account has at least 101 `unreservedConcurrency` to be able to set `reservedConcurrency` to 1. However, new created aws account usually has only 50 `unreservedConcurrency`. So removed `reservedConcurrency ` setting here and use another workaround to prevent concurrent lambda.